network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java

```diff
@@ -83,6 +83,7 @@ private void loadToCache(ResourceType resourceType, UUID networkUuid, int varian
             case DANGLING_LINE -> delegate.getDanglingLines(networkUuid, variantNum);
             case CONFIGURED_BUS -> delegate.getConfiguredBuses(networkUuid, variantNum);
             case TIE_LINE -> delegate.getTieLines(networkUuid, variantNum);
+            case GROUND -> delegate.getGrounds(networkUuid, variantNum);
```

Comment:
Should we have Ground in RESOURCE_TYPES_NEEDED_FOR_BUS_VIEW ?
To be confirmed

Reply:
Not required for now. LF, SecuAnalysis, Sensi and Shortcircuit don't use it. To be added later if needed.